src/core/reactor.ts

```diff
@@ -1976,7 +2009,7 @@ export class App extends Reactor {
     /**
      * Iterate over all reactions in the reaction queue and execute them.
      */
-    private _react() {
+    protected _react(): boolean {
```

Comment:
Needs a comment indicating the meaning of the returned value.

Reply:
The comment is added at this commit(02b36bd4ecab67b9fd656d5a872f04ff12c52f6d).